src/router.js

```diff
@@ -12,6 +12,11 @@ const routes = [
     name: 'home',
     component: Home,
   },
+  {
+    path: '*',
+    name: 'not-found',
+    component: () => import(/* webpackChunkName: "notfound" */ './views/NotFound'),
```

Comment:
This doesn't need to be a dynamic import IMO, as this component should be available at all times in case the user navigates to a non-existent page, we want the 404 page to already be loaded to prevent any further network activity and potential errors - so do this like the Home page above 👍